en/docs/security-announcements/cve-justifications/2025/CVE-2025-46392.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "CVE-2025-46392"
+category: security-announcements
+date: 
+---
+
+# CVE-2025-46392
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
+
+A Denial of Service attack through uncontrolled resource consumption by providing a crafted config file with malicious or recursive variable interpolation. [^1]
```

Comment:
```suggestion
A Denial of Service (DoS) vulnerability in Apache Commons Configuration 1.x which can be exploited by supplying a specially crafted configuration file containing malicious or recursive variable interpolation, leading to uncontrolled resource consumption.. [^1]
```

Reply:
Thanks for the suggestion. I do agree and I have committed the changes.